packages/driver/src/cy/commands/actions/type.ts

```diff
@@ -332,21 +339,32 @@ export default function (Commands, Cypress, cy, state, config) {
             updateTable(id, key, event, value)
           }
 
+          if (event.type === 'keydown') {
```

Comment:
Because you only push events onto this array when type === keydown, there's no need to check the type again later. So this could be simplified to

```
 if (event.type === 'keydown') {
  keydownEventTargets.push(event.target);
}

// then below...

// When a space key is pressed on another element, the click event should not be fired.
keydownEventTargets.includes(target)
```

removing the need to define `keydownFiredOnThisElement` at all.

Reply:
At first, I logged all events. But I realized that it could slow down Cypress a lot. So, the code became this way. 

Your code is much simpler. 

Thanks!